valhalla/baldr/rapidjson_utils.h

```diff
@@ -135,6 +135,15 @@ template <typename V> inline const rapidjson::Value& get_child(const V& v, const
   return *ptr;
 }
 
+template <typename V>
+inline const rapidjson::Value& get_child(const V& v, const char* source, const rapidjson::Value& t) {
```

Comment:
we use this in costing parsing, so that when the user doesnt provide costing options in their json we can have an empty dummy json object instead. this lets us simplify our logic in that we dont ahve to check if the object exists, we get back one regardless. instead of doing this we could have just added a bunch of ifs to the macros i made for costing options parsing